hypothesis-python/src/hypothesis/extra/django/_impl.py

```diff
@@ -104,7 +104,7 @@ def from_model(
         ):
             field_strategies[name] = from_field(field)
 
-    for field in field_strategies:
+    for field in field_strategies.copy():
```

Comment:
There is no need to copy here because we we return whenever a "pop" occurs, thus the dictionary size never changes between consecutive iterations